CHANGELOG.md

```diff
@@ -27,24 +45,14 @@ into
 Box(cornerStyle: .capsule)
 ```
 
-- Add `accessibilityFrameSize` to `AccessibilityElement` for manually specifying a size for the frame rendered by Voice Over.
+- Add `accessibilityFrameSize` to `AccessibilityElement` for manually specifying a size for the frame rendered by Voice Over. ([#144])
 
-### Removed
+- Add `Opacity` element for modifying the opacity of a wrapped element. ([#147])
```

Comment:
Thanks! Totally forgot this in my PR

Reply:
haha nw! I do all the time (and also forgot to ask for it in the code review)